typescript/infra/helm/liquidity-layer-relayers/templates/env-var-external-secret.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: liquidity-layer-env-var-external-secret
+  labels:
+    {{- include "abacus.labels" . | nindent 4 }}
+spec:
+  secretStoreRef:
+    name: {{ include "abacus.cluster-secret-store.name" . }}
+    kind: ClusterSecretStore
+  refreshInterval: "1h"
+  # The secret that will be created
+  target:
+    name: liquidity-layer-env-var-secret
+    template:
+      type: Opaque
+      metadata:
+        labels:
+          {{- include "abacus.labels" . | nindent 10 }}
+        annotations:
+          update-on-redeploy: "{{ now }}"
+      data:
+        GCP_SECRET_OVERRIDES_ENABLED: "true"
+        GCP_SECRET_OVERRIDE_ABACUS_{{ .Values.abacus.runEnv | upper }}_KEY_DEPLOYER: {{ print "'{{ .deployer_key | toString }}'" }}
```

Comment:
Do the circle & portal relayers use the deployer key?

Just wanna be cognizant of potential conflicts with testnet3 if we actually use the deployer key for a lot of long-living things on testnet2 & testnet3

Reply:
They do but i was hoping that this is an acceptable crutch for now. It really would be nice to have a global tx inclusion service, since that's all I care about here